### ISSUE TITLE
RSeQC TIN module - sample name cleaning

### ISSUE THREAD

**Issue by ewels** [ewels/MultiQC#1484]
_Originally posted by @guidohooiveld in https://github.com/ewels/MultiQC/issues/737#issuecomment-874317166_

---

Thanks Phil and Erik for creating the MultiQC RSeQC TIN submodule; much appreciated!

Earlier today I updated MutiQC to the latest development version, and ran it again on a map containing various QC output files, including TIN.
Et voila, the 2 columns (of which one is hidden) were indeed added to the General Statistics table. Nice & thanks!

One comment/question, though, regarding the sample names used for the TIN values in the General Statistics table: these are not the same as used for the other RSeQC modules. This makes the table 'less nice' and more difficult to read. See 1st screenshot below.

I think this is due to the fact that within the TIN "summary" file (the txt file `*out.summary.txt`) the full name of the BAM file is returned (used) by RSeQC (see its copied content below), which is then extracted (parsed) by the MultiQC TIN module, and subsequently used in the General Statistics table.

Therefore: would you have any suggestion to prevent this form happening? So that only the 'base name' is used in the table? Maybe by somehow using on-the-fly the function `fn_clean_sample_names`?
Note that I am not an expert on how to do this and it may be a too naive thought... but since the 'other' files seem to be correctly recognized and name cleaned (see 2nd screenshot), this may be feasible.

Thus, in summary: in the General Statistics table the full name present in the TIN  summary file (`*out.summary.txt`) is used (e.g. "P26-1-6h_Aligned.sortedByCoord.out.bam"), whereas just the use of only the sample ID (base name) "P26-1-6h" would be preferred.

Content TIN summary file (`P26-1-6h_Aligned.sortedByCoord.out.summary.txt`):
```
Bam_file	TIN(mean)	TIN(median)	TIN(stdev)
P26-1-6h_Aligned.sortedByCoord.out.bam	53.72327495737302	53.34221052273402	18.530355596890026

```
An example file is present in my previous post in this thread (https://github.com/ewels/MultiQC/issues/737#issuecomment-858673304).
	 

![image](https://user-images.githubusercontent.com/22979991/124501559-158eb500-ddc2-11eb-9646-c38fadb16985.png)

Below a screenshot of a map containing for a sample the output of STAR, but also RSeQC and Picard. All relevant files are nicely recognized by MultiQC, and their names are properly 'cleaned' when used in the MultiQC report. Hence my (naive) thought above...

![image](https://user-images.githubusercontent.com/22979991/124516704-9198f500-dde2-11eb-8c22-8a39101bc4e5.png)

**Comment by ewels** [comment]
@guidohooiveld - fixed in v1.12dev. If you [install the dev version](https://github.com/ewels/MultiQC#installation) it should now work for you as you expect.

Sorry for the inconvenience!

Phil

**Comment by guidohooiveld** [comment]
Thanks Phil for the prompt action taken! It is indeed working now as expected.



![image](https://user-images.githubusercontent.com/22979991/124666615-9925d000-deae-11eb-9e8b-6a8fa0772b73.png)
